modules/configuring-oidc-authentication.adoc

```diff
@@ -0,0 +1,236 @@
+
+By configuring Red Hat Single Sign-On on {productname}, you can create a seamless authentication integration between {productname} and other application platforms like {ocp}. 
+
+[id="configuring-red-hat-sso-using-config-tool"]
+=== Configuring the Red Hat Single Sign-On Operator for the {productname} Operator
+
+Use the following procedure to configure Red Hat Single Sign-On for the {productname} Operator on {ocp}. 
+
+.Prerequisites 
+
+* You have configured Red Hat Single Sign-On for OpenShift. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6/html-single/server_installation_and_configuration_guide/index#operator[Red Hat Single Sign-On Operator]. 
+* You have configured TLS/SSL for your {productname} deployment _and_ for Red Hat Single Sign-On. 
+* You have generated a single Certificate Authority (CA) and uploaded it to your Red Hat Single Sign-On Operator _and_ to your {productname} configuration.  
+* You are logged into your {ocp} cluster. 
+* You have installed the OpenShift CLI (`oc`). 
+
+.Procedure 
+
+. Upload a custom SSL/TLS certificate to your {ocp} deployment by running the following commands: 
+
+.. Base64 encode the content of a single TLS/SSL certificate by entering the following command:
++
+[source,terminal]
+----
+$ cat ssl.crt | base64 -w 0
+----
++
+.Example output
++
+[source,terminal]
+----
+LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURCVENDQWUyZ0F3SUJBZ0lKQUtsYW9WNHMrb1BtTUEw
+[truncated]
+----
+
+.. Create a Secret called `quay-enterprise-cert-secret` by entering the following command: 
++
+[source,terminal]
+----
+$ oc create create secret generic quay-enterprise-cert-secret --from-file=ca.crt=<path_to_certificate_file>
+----
+
+.. Mount the secret into the `qe` pod in `/conf/stack/extra_ca_certs`:
++
+[source,yaml]
+----
+apiVersion: v1
+data:
+  ssl.crt:  LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURCVENDQWUyZ0F3SUJBZ0lKQUtsYW9WNHMrb1BtTUE[truncated]
+kind: Secret
+metadata:
+  name: quay-enterprise-cert-secret
+type: Opaque
+----
+
+.. Edit the `quay-enterprise-app` deployment to add the following mount path under `volumeMounts`. This can be done from the {ocp} console or by using `oc edit`.
++
+[source,yaml]
+----
+    - mountPath: /conf/stack/extra_ca_certs
+      name: certvolume
+----
+
+.. Under the `volumes` section, add the `quay-enterprise-cert-secret` as a volume. For example:
++
+[source,yaml]
+----
+    - name: certvolume
+      secret:
+        defaultMode: 420
+        secretName: quay-enterprise-cert-secret
+----
+
+.. Save the manifest. The `Quay` pods should be rescheduled automatically. 
+
+. Log in to the {ocp} web console and navigate to *Operators* -> *Installed Operators* and click on *Red Hat Single Sign-On*. 
+
+. Click on the Red Hat Single Sign-On instance that you will be using for {productname} to access the administration console. 
+
+. Navigate to the Red Hat Single Sign-On *Admin Console*. 
```

Comment:
I can't find d Hat Single Sign-On Admin Console from *Installed Operators*.

How I get the Red Hat Single Sign-On *Admin Console* in OCP web console
 * access OCP web console and go to `network` --> `route`
 * choose  Red Hat Single Sign-On project in `Project` drop down list.
 * find Red Hat Single Sign-On *Admin Console* in `Routes` table

Reply:
If you 100% sure that you can find  Red Hat Single Sign-On *Admin Console* by following line 95 to 97, just ignore my above comment.